base/dates/accessors.jl

```diff
@@ -89,3 +72,68 @@ yearmonthday(dt::TimeType) = yearmonthday(days(dt))
 @vectorize_1arg TimeType yearmonth
 @vectorize_1arg TimeType monthday
 @vectorize_1arg TimeType yearmonthday
+
+
+# Documentation for exported accessors
+for func in (:year, :month)
+    name = string(func)
+    @eval begin
+        @doc """
+            $($name)(dt::TimeType) -> Int64
+
+        The $($name) of a `Date` or `DateTime` as a `Int64`.
```

Comment:
The original documentation was stated this way. I did a quick check on a 32-bit VM and this does seem accurate. I'll fully check all of these methods and report back.